### PR TITLE
Merge data from "created_by" to "authors" field

### PR DESCRIPTION
**Pull request by jonasdeluna**
Migrate data from the old created_by field to the new authors field holding multiple authors instead of one.

### CODE REVIEW

File: lego/apps/articles/fixtures/development_articles.yaml [hunk 1]
```diff
@@ -36,6 +36,7 @@
       - lorem
       - ipsum
       - sample-text
+    created_by_id: 1
```
Comment (LudvigHz) [inline lego/apps/articles/fixtures/development_articles.yaml:39]: Add the authors field here as well

File: lego/apps/articles/migrations/0012_migrate_authors.py [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+from django.conf import settings
+from django.db import migrations
+
+
+def merge_authors(apps, schema_editor):
+    Article = apps.get_model("articles", "Article")
+    for e in Article.objects.all():
+        if e.created_by != None:
+            e.authors.add(e.created_by)
+            e.save()
```
Comment (LudvigHz) [inline lego/apps/articles/migrations/0012_migrate_authors.py:12]: I just realized this doesn't make sense when `authors` is not empty. Might be an idea to check if `authors` is empty as well :) (In theory this should not happen, but it's a bad idea to depend on things going the way you assume)

**Review by LudvigHz — COMMENTED** [review]
Nice :ok_hand: 

Fix up the comments (and preferably also change the commit message to something that makes more sense) and we're good to go :100:

File: lego/apps/articles/migrations/0012_migrate_authors.py [hunk 3]
```diff
@@ -0,0 +1,24 @@
+# Generated by Django 4.0.9 on 2023-03-16 13:13
+
+from django.conf import settings
+from django.db import migrations
+
+
+def merge_authors(apps, schema_editor):
+    Article = apps.get_model("articles", "Article")
+    for e in Article.objects.all():
+        if e.created_by != None and e.authors == None:
```
Comment (eikhr) [inline lego/apps/articles/migrations/0012_migrate_authors.py:10]: Should't this be `e.authors != None`, or can you do `.add()` anyways?

Comment (jonasdeluna) [reply lego/apps/articles/migrations/0012_migrate_authors.py:10]: It seemed to run fine for me. From testing in shell, the field is None if there's no entries and adding still works.

**Review by LudvigHz — APPROVED** [review]

**Pull request merged**